### PR TITLE
Format code using 'cargo fmt'

### PR DESCRIPTION
**Pull request by Atul9**
License: I license past and future contributions under the dual MIT/Apache-2.0 license,
allowing licensees to chose either at their option.

### CODE REVIEW

File: src/jpeg/transform.rs [hunk 1]
```diff
@@ -4,51 +4,51 @@ Independent JPEG Group's libjpeg version 9a
...
+    acknowledge us.
+
+    Permission is NOT granted for the use of any IJG author's name or company name
+    in advertising or publicity relating to this software or products derived from
+    it.  This software may be referred to only as "the Independent JPEG Group's
+    software".
+
+    We specifically permit and encourage the use of this software as the basis of
+    commercial products, provided that all warranty or liability claims are
+    assumed by the product vendor.
```
Comment (HeroicKatora) [inline src/jpeg/transform.rs]: We can't let it alter the copyright not here.

File: src/jpeg/encoder.rs [hunk 2]
```diff
@@ -903,13 +898,23 @@ mod tests {
...
                 1, // density is in dpi
-                300u16.to_be_bytes()[0], 300u16.to_be_bytes()[1],
-                300u16.to_be_bytes()[0], 300u16.to_be_bytes()[1],
-                0, 0, // No thumbnail
+                300u16.to_be_bytes()[0],
+                300u16.to_be_bytes()[1],
+                300u16.to_be_bytes()[0],
+                300u16.to_be_bytes()[1],
+                0,
+                0, // No thumbnail
```
Comment (HeroicKatora) [inline src/jpeg/encoder.rs]: This does not improve the readability of the test.

File: src/buffer.rs [hunk 3]
```diff
@@ -1261,88 +1283,163 @@ mod test {
...
+        assert!(!image.copy_within(
+            Rect {
+                x: 1,
+                y: 1,
+                width: 4,
+                height: 4
+            },
+            0,
+            0
+        ));
```
Comment (HeroicKatora) [inline src/buffer.rs]: Now this has just gotten entirely unreadable.

File: src/dynimage.rs [hunk 4]
```diff
@@ -817,7 +812,9 @@ impl GenericImage for DynamicImage {
             DynamicImage::ImageBgr8(ref mut p) => p.put_pixel(x, y, pixel.to_bgr()),
             DynamicImage::ImageBgra8(ref mut p) => p.put_pixel(x, y, pixel.to_bgra()),
             DynamicImage::ImageLuma16(ref mut p) => p.put_pixel(x, y, pixel.to_luma().into_color()),
-            DynamicImage::ImageLumaA16(ref mut p) => p.put_pixel(x, y, pixel.to_luma_alpha().into_color()),
+            DynamicImage::ImageLumaA16(ref mut p) => {
+                p.put_pixel(x, y, pixel.to_luma_alpha().into_color())
+            }
```
Comment (HeroicKatora) [inline src/dynimage.rs]: Visual consistency does not exist in the dictionary of `cargo fmt`.

**Comment by Atul9** [comment]
@HeroicKatora ok, let's close this PR. Can you please add these above comments & other suggestions about cargo fmt in https://github.com/image-rs/organization/blob/master/CONTRIBUTING.md?

**Comment by HeroicKatora** [comment]
@Atul9 Sure, that sounds great. I'll ping you for wording.

**Pull request closed without merging**